x-pack/packages/ml/trained_models_utils/src/constants/trained_models.ts

```diff
@@ -284,3 +284,9 @@ export type InferenceAPIConfigResponse = {
     model?: string;
   };
 } & InferenceServiceSettings;
+
+export function isLocalModel(
```

Comment:
why not `isElasticModel`?

Reply:
to me Elastic Model is E5 and Elser, but people can also bring their own trained models that aren't Elastic owned. I'm fine with changing it to isElasticModel if you feel that's more appropriate.